src/Effects.js

```diff
@@ -57,6 +58,34 @@ module.exports = class Effects {
       }
     };
 
+    this.starz = {
+      starSizes: [10, 9, 8, 7, 6, 5, 4, 3, 2, 1],
+
+      draw: function() {
+        translate(200, 200);
+        let ordered = [];
+        for (let size of starSizes) {
+          ordered.push((size + frameCount / 80) % starSizes.length);
+        }
+        
+        ordered.sort((a, b) => b - a);
+        
+        for (let size of ordered) {
+          let color = lerpColorFromPalette(ceil(size) % 4);
+          push();
+          scale(size);
+          fill(color);
+          drawStar(0, 0, 30, 70, 5);
+          pop();
+        }
+      },
+
+      setup: function() {
```

Comment:
I don't think you need this. The canvas size and frame rate are the same for all background effects.